src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -68,6 +67,23 @@
 public class HttpConnection {
 
     private static final Logger logger = Logger.getLogger(HttpConnection.class.getCanonicalName());
+
+    private static final String USER_AGENT;
+
+    static {
+        String ua = "java-cloudant-http/unknown";
```

Comment:
Its the responsibility of the Version class to provide anything it wants to use as the UA string, which for our libs is a version string with some extra bits. When it's shared between sync-android, it will still be this as the default since its the default for the package, eg this is the java-cloudant-http so that's what it should be reported as. Perhaps a rename to getUserAgent? instead of getVersion?